packages/utils/src/ethereum.ts

```diff
@@ -64,94 +64,132 @@ const createEthereumChainInfo = <
   }),
   "Goerli Testnet": mapChainInfo({
     chain: goerli,
     axelarChainName: "Goerli Testnet",
     clientChainId: "Goerli Test Network",
+    color: "#454973",
+    relativeLogoUrl: "/networks/ethereum.svg",
   }),
   "Binance Smart Chain": mapChainInfo({
     chain: bsc,
     axelarChainName: "Binance Smart Chain",
     clientChainId: "Binance Smart Chain Mainnet",
+    color: "#f3b90c",
+    relativeLogoUrl: "/networks/binance.svg",
   }),
   "BSC Testnet": mapChainInfo({
     chain: bscTestnet,
     axelarChainName: "BSC Testnet",
     clientChainId: "Binance Smart Chain Testnet",
+    color: "#f3b90c",
+    relativeLogoUrl: "/networks/binance.svg",
   }),
   Polygon: mapChainInfo({
     chain: polygon,
     axelarChainName: "Polygon",
     clientChainId: "Polygon Mainnet",
+    color: "#8247E5",
+    relativeLogoUrl: "/networks/polygon.svg",
   }),
   Mumbai: mapChainInfo({
     chain: polygonMumbai,
     axelarChainName: "Mumbai",
     clientChainId: "Mumbai",
+    color: "#8247E5",
+    relativeLogoUrl: "/networks/polygon.svg",
   }),
   Moonbeam: mapChainInfo({
     chain: moonbeam,
     axelarChainName: "Moonbeam",
     clientChainId: "Moonbeam Mainnet",
+    color: "#FA047C",
+    relativeLogoUrl: "/networks/moonbeam.svg",
   }),
   "Moonbase Alpha": mapChainInfo({
     chain: moonbaseAlpha,
     axelarChainName: "Moonbase Alpha",
     clientChainId: "Moonbase Alpha",
+    color: "#FA047C",
+    relativeLogoUrl: "/networks/moonbeam.svg",
   }),
   Fantom: mapChainInfo({
     chain: fantom,
     axelarChainName: "Fantom",
     clientChainId: "Fantom Opera",
+    color: "#1b6cfc",
+    relativeLogoUrl: "/networks/fantom.svg",
   }),
   "Fantom Testnet": mapChainInfo({
     chain: fantomTestnet,
     axelarChainName: "Fantom Testnet",
     clientChainId: "Fantom Testnet",
+    color: "#1b6cfc",
+    relativeLogoUrl: "/networks/fantom.svg",
   }),
   "Avalanche Fuji Testnet": mapChainInfo({
     chain: avalancheFuji,
     axelarChainName: "Avalanche Fuji Testnet",
     clientChainId: "Avalanche Fuji Testnet",
+    color: "#E84142",
+    relativeLogoUrl: "/networks/avalanche.svg",
   }),
   Avalanche: mapChainInfo({
     chain: avalanche,
     axelarChainName: "Avalanche",
     clientChainId: "Avalanche C-Chain",
+    color: "#E84142",
+    relativeLogoUrl: "/networks/avalanche.svg",
   }),
   Arbitrum: mapChainInfo({
     chain: arbitrum,
     axelarChainName: "Arbitrum",
     clientChainId: "Arbitrum One",
+    color: "#3D4A6B",
+    relativeLogoUrl: "/networks/arbitrum.svg",
   }),
   Filecoin: mapChainInfo({
     chain: filecoin,
     axelarChainName: "Filecoin",
     clientChainId: "Filecoin - Mainnet",
+    color: "#0494fc",
+    relativeLogoUrl: "/networks/filecoin.svg",
   }),
   "Filecoin Hyperspace": mapChainInfo({
     chain: filecoinHyperspace,
     axelarChainName: "Filecoin Hyperspace",
     clientChainId: "Filecoin Hyperspace",
+    color: "#0494fc",
+    relativeLogoUrl: "/networks/filecoin.svg",
```

Comment:
A problem with this is it may not work on other apps, like mobile for instance. I wonder if there's a way to map these images within web package?

Reply:
My thinking here is that since it's an object, mobile can either overwrite the values or follow the convention and create a folder with the same relative path